AdjointPlugin13Metasurface.ipynb

```diff
@@ -1719,7 +1719,7 @@
   }
  ],
  "metadata": {
-  "description": "This notebook demonstrates how to perform inverse design optimization of a diffractive metasurface in Tidy3D FDTD.",
+  "description": "This notebook demonstrates how to perform inverse design optimization of a diffractive metasurface in Tidy3D FDTD using the adjoint plugin..",
```

Comment:
I know it's only metadata but I think a superfluous period slipped in at the end.